src/pages/community/vision-and-mission.md

```diff
@@ -0,0 +1,21 @@
+<!-- Put each sentence on a single line to make reviewing easier -->
+# OpenInfra Labs Vision and Mission
+                                                                     
+This page is to help you understand the ovearching purpose (vision) of the OpenInfra Labs,, and how we're bringing about that vision, which is our mission.
```

Comment:
nit: duplicate ,

Reply:
Also: Purpose and vision are distinctly different.

Purpose is why you exist, which generally is mission related.

Vision is what you see as the result of your work down the road. If you can think of an obtainable, but slightly scary/uncomfortable end result of success, then that might be considered a good vision if others can use it to help drive alignment.